.github/workflows/package.yml

```diff
@@ -55,6 +55,9 @@ jobs:
         with:
           ruby-version: 3.1.4
 
+      - name: Install GitHub CLI
+        run: sudo apt-get update && sudo apt-get install gh -y
+
```

Comment:
believe this can be removed:
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md